### PR TITLE
Sdk 95 parse urls correctly

### PR DESCRIPTION
**Pull request by fundthmcalculus**
https://github.com/trinsic-id/server/issues/219
https://github.com/trinsic-id/sdk/issues/95

### CODE REVIEW

File: dotnet/Source/Trinsic.Tests/Tests.cs [hunk 1]
```diff
@@ -17,6 +18,29 @@ public Tests(ITestOutputHelper testOutputHelper)
         {
             _testOutputHelper = testOutputHelper;
         }
+        
+        private static string __FILE__([CallerFilePath] string fileName = "") => fileName;
+
+        private string TestDataPath =>
+            Path.GetFullPath(Path.Join(__FILE__(), "..", "..", "..", "..", "devops", "testdata"));
```
Comment (tmarkovski) [inline dotnet/Source/Trinsic.Tests/Tests.cs:25]: Yikes. Is there a better of passing the path? Maybe from ENV?

Comment (fundthmcalculus) [reply dotnet/Source/Trinsic.Tests/Tests.cs:25]: Possibly, but then we require environment variables to be known and set for testing. I'd rather compile it in so it's known. Also this eliminates `cwd` problems.

**Review by tmarkovski — APPROVED** [review]

File: dotnet/Source/Trinsic.Tests/Tests.cs (hunk 1 above)
Comment (tmarkovski) [reply dotnet/Source/Trinsic.Tests/Tests.cs:25]: We already require them for AppConfig Conn String. Can you instead add the files in the csproj with the test data raference?

```
<Include Path="../../../../..../testdata/file.jsonld" CopyToOutput=true />
```

or whatever the correct syntax is?

Comment (fundthmcalculus) [reply dotnet/Source/Trinsic.Tests/Tests.cs:25]: I can do that for C#, most other languages don't have that kind of build system. :/

**Pull request merged**